metadata-ingestion/source_docs/clickhouse.md

```diff
@@ -0,0 +1,64 @@
+# Clickhouse
+
+For context on getting started with ingestion, check out our [metadata ingestion guide](../README.md).
+
+## Setup
+
+To install this plugin, run `pip install 'acryl-datahub[clickhouse]'`.
+
+## Capabilities
+
+This plugin extracts the following:
+
+- Metadata for tables, views and materialized views
```

Comment:
I'm not sure if I saw any queries for views/materialized views in the logic. Are these included by default when calling `show tables from {database}` ?